win32ss/gdi/ntgdi/freetype.c

```diff
@@ -160,6 +160,28 @@ static const CHARSETINFO g_FontTci[MAXTCIINDEX] =
     { SYMBOL_CHARSET, CP_SYMBOL, {{0,0,0,0},{FS_SYMBOL,0}} }
 };
 
+BYTE FASTCALL IntCharSetFromCodePage(UINT uCodePage)
```

Comment:
I would suggest to add a comment that refers to gdi/gdi32/objects/font.c!TranslateCharsetInfo() function, in the  TCI_SRCCODEPAGE case.

Reply:
What like?

Reply:
OK.